### DISCUSSION TITLE
Hola! Alguien tiene espacio en su equipo? o podemos crear uno nuevo

### DISCUSSION THREAD

**Discussion by nicleor** [Launch-X-Latam/MisionFrontEnd#575] [Team Posts]
Hola busco un equipo para el hackaton o podemos hacer uno nuevo.

Este es mi correo: lx02649@innovaccion.mx

**Comment by Aldo-Luis-Medina-Resendiz** [comment]
Va que va, yo también busco a alguien el mío es lx14981@innovaccion.mx

**Comment by Dariopg** [comment]
Hola como están? Yo también estoy huérfano, quien me adopta para un equipo? lx06073@innovaccion.mx

**Comment by Caml02** [comment]
Me interesa,  ***@***.*** agregame jeje


El sáb, 19 nov 2022 a la(s) 15:45, Dariopg ***@***.***)
escribió:

> Hola como están? Yo también estoy huérfano, quien me adopta para un
> equipo? ***@***.***
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/Launch-X-Latam/teams/mision-backend/discussions/112/comments/2>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVIGTWNP5AXKP5RSS3PSZJTWJFJ7VANCNFSM6AAAAAASFQJPBY>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 


*Asesor Financiero  *

*Cristian A. Morales L.**_____________________________*
*(662) - 447- 6685*

*Todos Los Bancos En Un Solo Lugar*

**Comment by EWRC01** [comment]
Hola buenas igual busco equipo para el hackaton para que me adopten

Este es mi correo: ***@***.***
Usuario de git: EWRC01
Portafolio: ewrc01.github.io


Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by NotoriuxX** [comment]
Hola igual ando buscando equipo, podríamos juntarnos los 3

**Comment by PetuchoDeveloper** [comment]
Hola, yo también estoy huerfanito de equipo, pueden contactarme por Teams
lx15053@innovaccion.mx
No se mucho, pero estoy dispuesto a aprender :)

**Comment by denissegutierrez** [comment]
Hola nosotros, te adoptamos 😊

**Comment by denissegutierrez** [comment]
Listo, solo confírmame en Teams que estás agregado

**Comment by AlexGeovanni** [comment]
ya estan completos?

**Comment by denissegutierrez** [comment]
Esperamos confirmación pero sí

**Comment by PetuchoDeveloper** [comment]
Si, fui agregado muchas gracias :)

**Comment by Dariopg** [comment]
> Esperamos confirmación pero si

Tendrás un lugar para mí 😔😁

**Comment by ivnarnd** [comment]
hola a todos, estoy disponible también por si alguien necesita!
lx06643@innovaccion.mx es mi correo

**Comment by NotoriuxX** [comment]
Hola estoy disponible igualmente, mi correo es lx08405@innovaccion.mx

**Comment by ivnarnd** [comment]
buenas noches, yo tambien estoy disponible para formar equipo, mi correo es lx06643@innovaccion.mx

**Comment by Liz0798** [comment]
Hola, yo tambien estoy disponible :( mi correo es lx00636@innovaccion.mx

**Comment by NoirOrange** [comment]
@Liz0798 Hola, aún estás disponible?
